apps/demos/Demos/Common/PopupAndNotificationsOverview/Vue/App.vue

```diff
@@ -104,8 +104,8 @@ import { housesSource } from './data.ts';
 
 const ADD_TO_FAVORITES = 'Add to Favorites';
 const REMOVE_FROM_FAVORITES = 'Remove from Favorites';
-const houses = ref(housesSource);
-const currentHouse = ref(housesSource[0]);
+const houses = ref<Record<string, any>[]>(housesSource);
```

Comment:
unknown is better then any
```suggestion
const houses = ref<Record<string, unknown>[]>(housesSource);
```

Reply:
unknown not work there is house.Agent.Phone